custom-cakes/index.html

```diff
@@ -4,10 +4,70 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./reset/reset.css">
+  <link rel="stylesheet" href="./common/reset.css">
+  <link rel="stylesheet" href="./common/fonts.css">
+
+  <link rel="stylesheet" href="./modules/container/container.css">
+  <link rel="stylesheet" href="./modules/logo/logo.css">
+  <link rel="stylesheet" href="./modules/secondary-navigation/secondary-navigation.css">
+  <link rel="stylesheet" href="./modules/secondary-navigation/__list/secondary-navigation__list.css">
+  <link rel="stylesheet" href="./modules/secondary-navigation/__item/secondary-navigation__item.css">
+  <link rel="stylesheet" href="./modules/secondary-navigation/__link/secondary-navigation__link.css">
+  <link rel="stylesheet" href="./modules/social-list-secondary/social-list-secondary.css">
+  <link rel="stylesheet" href="./modules/social-list-secondary/__item/social-list-secondary__item.css">
+  <link rel="stylesheet" href="./modules/social-list-secondary/__icon-instagram/social-list-secondary__icon-instagram.css">
+  <link rel="stylesheet" href="./modules/social-list-secondary/__icon-vk/social-list-secondary__icon-vk.css">
+
+  <link rel="stylesheet" href="./modules/footer/footer.css">
+  <link rel="stylesheet" href="./modules/footer/__inner-footer/footer__inner-footer.css">
+  <link rel="stylesheet" href="./modules/footer/__header/footer__header.css">
+  <link rel="stylesheet" href="./modules/footer/__container/footer__container.css">
+  <link rel="stylesheet" href="./modules/footer/__copyright/footer__copyright.css">
+  <link rel="stylesheet" href="./modules/footer/__navigation/footer__navigation.css">
+  <link rel="stylesheet" href="./modules/footer/__social-list/footer__social-list.css">
+  <link rel="stylesheet" href="./modules/footer/__social-item/footer__social-item.css">
+  <link rel="stylesheet" href="./modules/footer/__social-item/footer__social-link.css">
   <title>Cakes</title>
 </head>
 <body>
-  <h1>Hello world!</h1>
+  <footer class="footer">
+    <div class="container footer__container">
```

Comment:
the footer should be placed inside the footer container not outside

Reply:
I supposed .footer__container to be .inner container. So I added additional inner section named "footer__top" to be the container I originally wanted to be.